charactersheet/charactersheet/templates/character/index.tmpl.html

```diff
@@ -218,8 +218,7 @@
     <div class="col-sm-8 col-lg-9 col-xs-12 col-padded">
       <div class="h3">
         Armor<br />
-        <small>Your Armor doesn't automatically influence your
-        Armor Class...yet.</small>
+        <small>Class calculations don't include Features or Feats.  Please make adjustments to AC in the adj number input.</small>
```

Comment:
My only argument to that is that the list of what it doesn't include is infinite, while what it does include is finite.